avocado/core/parameters.py

```diff
@@ -265,5 +262,5 @@ def iteritems(self):
         which generates lots of duplicate entries due to inherited values.
         """
         for leaf in self._leaves:
-            for key, value in iteritems(leaf.environment):
+            for key, value in iter(leaf.environment.items()):
```

Comment:
Just passing to say "do you really need the iter operator here?"

Reply:
No, this is a mistake (leftover from a previous iteration).  Thanks for spotting it.